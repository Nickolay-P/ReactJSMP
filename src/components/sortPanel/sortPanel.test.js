

import React from 'react';
import {shallow} from 'enzyme'
import {SortPanel} from './SortPanel.js';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

it('SortPanel rendered correctly', () => {
  const AppComponent = shallow(
    <SortPanel />
  );
  expect(AppComponent).toMatchSnapshot();
});