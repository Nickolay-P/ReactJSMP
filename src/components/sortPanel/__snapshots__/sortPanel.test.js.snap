// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortPanel rendered correctly 1`] = `ShallowWrapper {}`;
